source/_integrations/enphase_envoy.markdown

```diff
@@ -36,6 +36,8 @@ This integration will offer various sensors depending on the configuration of yo
 - Historical energy production & consumption
 - Power production per-inverter
 
+Production and consumption sensors for each phase are available for Envoy-metered with installed and configured current transformers (CT) on more than 1 phase.
```

Comment:
```suggestion
Production and consumption sensors for each phase are available for Envoy S Metered with installed and configured current transformers (CT) on more than 1 phase.
```
On their website, I found Envoy S Metered and Envoy S Metered Kit, without a dash. Is this the product you're referring to here?

Reply:
Yes. And since they renamed it to IQ Gateway, [IQ Gateway Metered](https://enphase.com/en-eu/store/communication/iq-gateway-metered) as well. You find still both names in use.

Reply:
Oh, I see. In that case, would it be worth adding both names?

Reply:
We could do Envoy S Metered / IQ Gateway metered.

Reply:
sounds good!